checkin/templates/checkin/profile_internal.html

```diff
@@ -0,0 +1,114 @@
+{% load html_tags %}
+{% load checkin_tags %}
+{% load static %}
+{% load thumbnail %}
+{% load i18n %}
+
+
+<div class="ui container">
+    <div class="ui image" data-tooltip="{% trans "Change picture" %}" data-position="right center">
+        {% if image %}
+            <img {% thumbnail image "150x150" crop="center" as image_thumb %}
+                src="{{ image_thumb.url }}"
+            {% empty %}
+                src="{{ image.url }}"
+            {% endthumbnail %}
+                id="profile-pic" alt="{% trans "profile picture"|capfirst %}"
+            />
+        {% else %}
+            <img id="profile-pic" src="{% static 'checkin/img/placeholder_profile.png' %}"
+                 alt="{% trans "Placeholder image" %}"
+            />
+        {% endif %}
+    </div>
+    <div class="ui container">
+        <form id="image-form" class="ui form" method="POST" action="{% url 'profile_picture' %}"
+              enctype="multipart/form-data">
+            {% csrf_token %}
+            <input id="input-image" type="file" name="image" accept="image/jpeg, image/png, image/svg" hidden/>
+        </form>
+    </div>
+</div>
+
+{% comment %}
```

Comment:
Just a suggestion, it would be nice to have an optional note to the comment about what the code is doing,  e.g. "Card registration and skills" or something similar.

https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#comment